cattle/compute/__init__.py

```diff
@@ -63,6 +47,27 @@ def instance_deactivate(self, req=None, instanceHostMap=None,
                                                         progress)
         )
 
+    def get_instance_host_from_map(self, instanceHostMap=None):
+        instance = instanceHostMap.instance
+        host = instanceHostMap.host
+
+        try:
+            host.clusterConnection = \
+                instanceHostMap.data.fields['clusterConnection']
+            log.debug('clusterConnection = %s', host.clusterConnection)
+            host.caCrt = \
+                instanceHostMap.data.fields['caCrt']
+            host.clientCrt = \
+                instanceHostMap.data.fields['clientCrt']
+            host.clientKey = \
+                instanceHostMap.data.fields['clientKey']
+            log.debug('caCrt=%s clientCrt=%s clientKey=%s',
+                      host.caCrt, host.clientCrt, host.clientKey)
+        except (KeyError, AttributeError):
+            pass
```

Comment:
I guess right now, I'm allowing for 3 different modes of operation.  If the KeyError/AttributeError is thrown because 'clusterConnection' isn't there, this is still a valid case.  This is just the case when we're deploying to a specific host rather than a cluster.
If 'clusterConnection' is there but not any of the cert info (this is a valid case for the time being while we're still using the current cattle server without the SSL stuff).  This just deploys to a non-SSL enabled cluster.